tests/rbd_mirror/test_rbd_mirror_daemon_status.py

```diff
@@ -0,0 +1,51 @@
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """
+    --> Deploy two clusters using Cephadm
+    --> Configures RBD Mirroring
+    --> Check the mirror status on primary and secoundary clusters
+    --> Check the rbd-daemons on cluster
```

Comment:
Could we follow the docString guidelines mentioned in document? https://peps.python.org/pep-0257/

Reply:
Thanks,
Updated all the changes.